config/initializers/localeapp.rb

```diff
@@ -0,0 +1,6 @@
+require 'localeapp'
+require 'localeapp/rails'
+
+Localeapp.configure do |config|
+  config.api_key = ENV['LOCALEAPP_KEY']
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.